ISSUE_TEMPLATE.md

```diff
@@ -21,13 +22,17 @@
 
 ...
 
-### Reproduction steps / Stream URLs to test
+### Reproduction steps / Explicit stream URLs to test
 
 1. ...
 2. ...
 3. ...
 
-### Environment details (operating system, python version, etc.)
+### Environment details
+
+Operating system: ...
```

Comment:
Operating system version would be a good addition in case people are using XP or something.